include/openthread/platform/radio.h

```diff
@@ -544,6 +556,16 @@ extern void otPlatRadioEnergyScanDone(otInstance *aInstance, int8_t aEnergyScanM
  */
 int8_t otPlatRadioGetReceiveSensitivity(otInstance *aInstance);
 
+/**
+ * The radio driver calls this method to notify OpenThread to process transmit security for the frame,
+ * this happens when the frame includes Header IE(s) that were updated before transmission.
+ *
+ * @param[in]  aInstance   The OpenThread instance structure.
+ * @param[in]  aFrame      The radio frame which neeeds to process transmit security.
+ *
+ */
+extern void otPlatRadioFrameUpdated(otInstance *aInstance, otRadioFrame *aFrame);
```

Comment:
A comment should include an explanation/warning that this function may be called from ISR context.

Reply:
Added, thanks.